src/new-client/src/App/Header/theme-switcher/ThemeSwitcher.tsx

```diff
@@ -1,13 +1,19 @@
-import React, { useState } from "react"
-import Switch, { ReactSwitchProps } from "react-switch"
-import { ThemeName, Theme, useTheme } from "theme"
-import styled, { withTheme } from "styled-components/macro"
+import { useState } from "react"
+import SwitchO, { ReactSwitchProps } from "react-switch"
+import { ThemeName, Theme, useTheme } from "@/theme"
+import { withTheme } from "styled-components"
+import styled from "styled-components"
 import DarkThemeIcon from "./DarkThemeIcon"
 import LightThemeIcon from "./LightThemeIcon"
 
 export interface ThemeStorageSwitchProps extends Partial<ReactSwitchProps> {}
 const iconSize = 18
 
+// TODO: Get rid of this library or open an issue/PR to fix their build
+const Switch: typeof SwitchO = (SwitchO as any).default
+  ? (SwitchO as any).default
+  : SwitchO
```

Comment:
Ugh, grim. Is this just for the theme switcher? Let's put it on the backlog to drop this library if we can. It's probably overkill anyway.

Reply:
:100: Agree! I was going to mention that in today's stand up